configurations/arm/amazon2.yaml

```diff
@@ -0,0 +1,7 @@
+ami_id_db_scylla: 'ami-06b4c3e5d2605128e'  # Amazon Linux 2 LTS Arm64 AMI 2.0.20220606.1 arm64 HVM gp2
+region_name: 'eu-west-1'
+instance_type_db: 'im4gn.xlarge'
```

Comment:
in the original file, the instance type is `i3.large`.. is this the equivalent for ARM?

Reply:
Yes. For reference, you can take a look at [artifacts-ami-test-arm](https://jenkins.scylladb.com/view/master/job/scylla-master/job/artifacts/job/artifacts-ami-test-arm/76/parameters/). The x86 one uses `i3.large`, and the arm one uses `im4gn.xlarge` instead.

Reply:
yes, i saw that... that's why i'm asking... wouldn't the parallel of that `i3.large` be `im4gn.large` ?

Reply:
other than this question, and failed mocked_tests it LGTM

Reply:
It's what roy recommended back when we first started to support arm
https://github.com/scylladb/scylla-pkg/issues/2744#issuecomment-1034782347
@roydahan Thoughts?